chompfile.toml

```diff
@@ -10,7 +10,7 @@ deps = ['dist/cli.js', 'docs.md']
 [[task]]
 target = 'dist/cli.js'
 deps = ['src/**/*.ts', 'npm:install']
-run = 'esbuild src/cli.ts --bundle --platform=node --external:@jspm/generator --external:ora --external:picocolors --external:@babel/core --format=esm --outfile=$TARGET'
+run = 'esbuild src/cli.ts --bundle --platform=node --external:fsevents --external:@jspm/generator --external:ora --external:picocolors --external:@babel/core --format=esm --outfile=$TARGET'
```

Comment:
We should move to a self-build at some point if we can...!

Reply:
Regarding this, i have some doubts. On what's the best possible way

- Show we just mark `rollup` as a `peerDependency` and `options`. Since users who don't use `jspm build` don't need to install `rollup` ?
- If we define `rollup` as direct dependency instead of `peer` or `optional`. What happens if users install their own version of `rollup` while defining the `config.mjs` file. Which version of rollup get's picked. The one from the cli or the one coming from the users project when they run `jspm build` ?

Reply:
Here is one from tests, where users don't need to install rollup. And just export the config alone. 
https://github.com/jspm/jspm-cli/blob/fa9c2d676710ca2b3c55eac5ecca9afc68539e1e/test/fixtures/scenario_build_app/rollup-config.mjs